src/hotspot/share/opto/loopnode.cpp

```diff
@@ -3678,6 +3678,13 @@ void PhaseIdealLoop::replace_parallel_iv(IdealLoopTree *loop) {
         !incr2->in(2)->is_Con())
       continue;
```

Comment:
Since you touch near code, please update style in these lines too to use `{}`.

Reply:
The whole function is formatted.